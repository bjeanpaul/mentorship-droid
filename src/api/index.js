export * from 'src/api/schemas';
export * from 'src/api/profiles';
export * from 'src/api/schedule';
export * from 'src/api/activities';
export * from 'src/api/categories';
export * from 'src/api/events';
export * from 'src/api/errors';
export * from 'src/api/sync';
export * from 'src/api/calls';
export * from 'src/api/notificationSettings';
export * from 'src/api/callNotes';
export * from 'src/api/messages';
export * from 'src/api/imageUrl';
export { default as imageUrl } from 'src/api/imageUrl';
