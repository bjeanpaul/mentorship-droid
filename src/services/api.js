const HOSTNAME = 'http://localhost:8000/mentor'

export function callAPI(endpoint, hash) {

  return fetch(`${HOSTNAME}${endpoint}`, {
    headers: {
      'Authorization': `Basic ${hash}`
    }
  })
  .then(response => response.json().then(json => ({ json, response })))
  .then(({json, response}) => {
    if (response.ok === false) {
      return Promise.reject(json)
    }
    return json
  })

}
