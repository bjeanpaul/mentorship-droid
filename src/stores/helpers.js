import moment from 'moment';
import { values, isUndefined, sortBy, filter, first } from 'lodash';


export const getAuthUserProfile = ({
  auth: { profileId },
  entities: { profiles },
}) => !isUndefined(profileId)
  ? profiles[profileId]
  : null;


export const getContext = state => {
  const { auth: { auth } } = state;

  return {
    auth,
    profile: getAuthUserProfile(state),
  };
};


// TODO sort?
export const getCategories = ({ entities: { categories } }) => values(categories);


// TODO sort?
export const getCategoryActivities = ({
  entities: { activities },
}, targetCategoryId) => (
  values(activities)
    .filter(({ category }) => category === targetCategoryId));


export const getCategory = ({ entities: { categories } }, id) => categories[id];


export const getActivity = ({ entities: { activities } }, id) => activities[id];


export const getScheduledCalls = ({ entities: { scheduledCalls } }) => values(scheduledCalls);


export const getScheduledCall = ({ entities: { scheduledCalls } }, id) => scheduledCalls[id];


export const getScheduledCallActivity = (state, id) => {
  const scheduledCall = getScheduledCall(state, id);
  return scheduledCall && scheduledCall.activity && getActivity(state, scheduledCall.activity);
};


export const getEvents = ({ entities: { events } }) => values(events);


export const getActivityCallNotes = ({
  entities: { callNotes },
}, activityId) => (
  filter(callNotes, { callActivity: activityId }));


export const getCall = ({ entities: { calls } }, id) => calls[id];


export const getCallNote = ({ entities: { callNotes } }, id) => callNotes[id];


export const getNextScheduledCall = (state, predicate = {}, time = Date.now()) => {
  let res = sortBy(getScheduledCalls(state), ({ callTime }) => callTime);
  res = res.filter(({ callTime }) => moment(callTime).isSameOrAfter(time));
  res = filter(res, predicate);
  return first(res);
};
