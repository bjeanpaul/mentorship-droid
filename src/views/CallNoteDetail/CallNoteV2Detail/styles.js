import { StyleSheet } from 'react-native';
import colors from 'src/constants/colors';

const LARGE_CIRCLE_SIZE = 72;

export default StyleSheet.create({
  scrollView: {
    padding: 24,
  },
  titleContainer: {
    borderColor: colors.CALL_NOTE_DETAIL_TITLE_BORDER,
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderStyle: 'solid',
    paddingBottom: 19,
    marginBottom: 18,
  },
  section: {
    marginBottom: 45,
  },
  sectionBody: {
  },
  sectionBodyActivityItems: {
    flex: 1,
    flexDirection: 'row',
    alignItems: 'center',
  },
  activityImage: {
    width: 72,
    height: 72,
    marginRight: 16,
    marginBottom: 12,
  },
  menteeStateImage: {
    alignSelf: 'center',
    marginRight: 20,
    borderWidth: 4,
    borderRadius: LARGE_CIRCLE_SIZE,
    width: LARGE_CIRCLE_SIZE,
    height: LARGE_CIRCLE_SIZE,
    borderColor: colors.CALL_NOTE_MOOD_IMAGE_SELECTED_BORDER,
  },
  activityObjective: {
    flex: 1,
    paddingTop: 0,
    paddingBottom: 0,
  },
  activityHelpfulText: {
    textAlign: 'center',
  },
  sectionMoodBody: {
    flexDirection: 'row',
    alignItems: 'center',
  },
});
