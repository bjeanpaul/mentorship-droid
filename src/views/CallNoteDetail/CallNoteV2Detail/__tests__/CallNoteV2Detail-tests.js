import React from 'react';
import { noop } from 'lodash';

import { imageUrl } from 'src/api';
import CallNoteV2Detail from 'src/views/CallNoteDetail/CallNoteV2Detail';
import { uidEquals, fakeCallNoteV2, fakeCall, fakeActivity } from 'app/scripts/helpers';

describe('CallNoteV2Detail', () => {
  const createComponent = (props = {}) => (
    <CallNoteV2Detail
      callNote={fakeCallNoteV2({ call: 2 })}
      call={fakeCall({
        id: 2,
        activity: 6,
      })}
      activity={fakeActivity({ id: 6 })}
      onBackPress={noop}
      {...props}
    />
  );

  it('should render', () => {
    const el = render(createComponent());
    expect(el).toMatchSnapshot();
  });

  it('should exclude the activity views if they are undefined', () => {
    const el = render(createComponent({ activity: void 0 }));
    expect(el).toMatchSnapshot();
  });

  it('should exclude the activity icon if it does not exist', () => {
    const el = render(createComponent({
      activity: fakeActivity({
        icon: imageUrl(null),
      }),
    }));
    expect(el).toMatchSnapshot();
  });

  it('should call onBackPress', () => {
    const onBackPress = jest.fn();
    const el = shallow(createComponent({ onBackPress }));

    el.findWhere(uidEquals('back'))
      .simulate('press');

    expect(onBackPress).toBeCalled();
  });
});
