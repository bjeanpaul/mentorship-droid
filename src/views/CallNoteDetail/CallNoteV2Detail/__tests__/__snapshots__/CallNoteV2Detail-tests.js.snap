exports[`CallNoteV2Detail should exclude the activity icon if it does not exist 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Tue, Sep 27, 10:09 am
    </Text>
    <TouchableNativeFeedback
      onPress={[Function]}>
      <View
        style={
          Array [
            Object {
              "position": "absolute",
              "top": 24,
            },
            Object {},
          ]
        }>
        <Image
          source={1}
          style={
            Array [
              Object {
                "height": 24,
                "width": 24,
              },
              Object {},
            ]
          } />
      </View>
    </TouchableNativeFeedback>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HOW DID THE CALL GO?
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          We completed it
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          ACTIVITY
        </Text>
      </View>
      <View
        style={Object {}}>
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
            }
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#003035",
                  "fontFamily": "Brandon Text",
                },
                Object {
                  "fontSize": 13,
                  "textAlign": "center",
                },
                Array [
                  Object {
                    "fontSize": 18,
                    "marginBottom": 12,
                    "textAlign": "left",
                  },
                  Object {
                    "flex": 1,
                    "paddingBottom": 0,
                    "paddingTop": 0,
                  },
                ],
              ]
            }>
            Let us make a Snowflake to Learn about Connection
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          OBJECTIVE
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Objective was achieved
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          ACTIVITY RATING
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Good
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          dzdfdszf
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEE’S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
              "borderColor": "#f77040",
              "borderRadius": 72,
              "borderWidth": 4,
              "height": 72,
              "marginRight": 20,
              "width": 72,
            }
          } />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Happy
        </Text>
      </View>
    </View>
  </ScrollView>
</View>
`;

exports[`CallNoteV2Detail should exclude the activity views if they are undefined 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Tue, Sep 27, 10:09 am
    </Text>
    <TouchableNativeFeedback
      onPress={[Function]}>
      <View
        style={
          Array [
            Object {
              "position": "absolute",
              "top": 24,
            },
            Object {},
          ]
        }>
        <Image
          source={1}
          style={
            Array [
              Object {
                "height": 24,
                "width": 24,
              },
              Object {},
            ]
          } />
      </View>
    </TouchableNativeFeedback>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HOW DID THE CALL GO?
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          We completed it
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          OBJECTIVE
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Objective was achieved
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          ACTIVITY RATING
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Good
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          dzdfdszf
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEE’S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
              "borderColor": "#f77040",
              "borderRadius": 72,
              "borderWidth": 4,
              "height": 72,
              "marginRight": 20,
              "width": 72,
            }
          } />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Happy
        </Text>
      </View>
    </View>
  </ScrollView>
</View>
`;

exports[`CallNoteV2Detail should render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Tue, Sep 27, 10:09 am
    </Text>
    <TouchableNativeFeedback
      onPress={[Function]}>
      <View
        style={
          Array [
            Object {
              "position": "absolute",
              "top": 24,
            },
            Object {},
          ]
        }>
        <Image
          source={1}
          style={
            Array [
              Object {
                "height": 24,
                "width": 24,
              },
              Object {},
            ]
          } />
      </View>
    </TouchableNativeFeedback>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HOW DID THE CALL GO?
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          We completed it
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          ACTIVITY
        </Text>
      </View>
      <View
        style={Object {}}>
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
            }
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#003035",
                  "fontFamily": "Brandon Text",
                },
                Object {
                  "fontSize": 13,
                  "textAlign": "center",
                },
                Array [
                  Object {
                    "fontSize": 18,
                    "marginBottom": 12,
                    "textAlign": "left",
                  },
                  Object {
                    "flex": 1,
                    "paddingBottom": 0,
                    "paddingTop": 0,
                  },
                ],
              ]
            }>
            Let us make a Snowflake to Learn about Connection
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          OBJECTIVE
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Objective was achieved
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          ACTIVITY RATING
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Good
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <View
        style={Object {}}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          dzdfdszf
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "marginBottom": 18,
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEE’S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
              "borderColor": "#f77040",
              "borderRadius": 72,
              "borderWidth": 4,
              "height": 72,
              "marginRight": 20,
              "width": 72,
            }
          } />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "fontSize": 18,
                "marginBottom": 12,
                "textAlign": "left",
              },
            ]
          }>
          Happy
        </Text>
      </View>
    </View>
  </ScrollView>
</View>
`;
